### DISCUSSION TITLE
我是神经病

### DISCUSSION THREAD

**Discussion by Abjust** [daizihan233/HanBot#43] [General]
~~我是神经病，我是我是神经病啊~~
~~我是神经病，我啊为钱不要命~~
~~我是神经病，我没事就撅个腚~~
~~我是神经病，我一点都没有人性~~

**Comment by Abjust** [comment]
~~《认清自我》~~（大嘘）

**Comment by SM-N770FZKGTGY** [comment]
> ~~我是神经病，我是我是神经病啊~~
> ~~我是神经病，我啊为钱不要命~~
> ~~我是神经病，我没事就撅个腚~~
> ~~我是神经病，我一点都没有人性~~

ccc
